coalib/results/Diff.py

```diff
@@ -508,3 +510,69 @@ def remove(self, range):
         :param range: The range to delete.
         """
         self.replace(range, '')
+
+
+def coala_unified_diff(a, b, fromfile='', tofile='', fromfiledate='',
+                       tofiledate='', n=3, lineterm='\n'):
+    r"""
+    This code is adapted from python's difflib library:
+    https://svn.python.org/projects/python/trunk/Lib/difflib.py
```

Comment:
which revision did you use as a base?
Please see https://github.com/coala/coala/issues/2006#issuecomment-261844477